cypress/e2e/pages/topicPage/tests.js

```diff
@@ -31,69 +31,73 @@ export default ({ service, pageType, variant }) => {
             otherVariant = variant === 'simp' ? 'trad' : 'simp';
           }
         }
-
         // Gets the topic page data for all the tests
-        cy.getPageData({
-          service,
-          pageType: 'topic',
-          variant,
-        }).then(({ body }) => {
-          topicTitle = body.data.title;
-          variantTopicId = body.data.variantTopicId;
-          pageCount = body.data.pageCount;
-          numberOfItems = body.data.curations[0].summaries.length;
-          firstItemHeadline = body.data.curations[0].summaries[0].title;
-          messageBanner = body.data.curations.find(
-            curation =>
-              curation.visualProminence === 'NORMAL' &&
-              curation.visualStyle === 'BANNER',
-          );
-        });
+        const data = await cy.getPageDataFromWindow();
```

Comment:
@karinathomas I refactored this bit to use async/await instead of promise.then .. I find this easier to read and maintain, but let me know if you disagree or if there are implications I didn't think of.